package.json

```diff
@@ -10,12 +10,14 @@
     "build:clean": "rm -rf build",
     "build": "yarn build:clean && yarn tsc",
     "lint": "eslint . --ext .ts",
-    "test": "jest"
+    "test": "jest --forceExit --detectOpenHandles -i"
```

Comment:
Why you added this flags ?

Reply:
The flag `--forceExit` force Jest to exit after all tests have completed running and flag `--detectOpenHandles` works with it in order for Jest to exit to potentially track down the reason. The flag `-i` is often used for debugging, it run all tests serially in the current process, rather than creating a worker pool of child processes that run tests.